src/commands/gate/api.ts

```diff
@@ -0,0 +1,39 @@
+import {Writable} from 'stream'
+
+import {AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios'
+
+import {getRequestBuilder} from '../../helpers/utils'
+
+import {Payload} from './interfaces'
+
+export const evaluateGateRules = (request: (args: AxiosRequestConfig) => AxiosPromise<AxiosResponse>) => async (
+  evaluateRequest: Payload,
+  write: Writable['write']
+) => {
+  // Converting the payload to the format expected from Rapid
```

Comment:
nitpick: I would avoid using references to services / technologies used by datadog

Reply:
Oh, you're right, changed!